profile/README.md

```diff
@@ -23,21 +22,13 @@ Don't forget to take a look at [`kwctl`](https://github.com/kubewarden/kwctl), o
 ## Writing policies 📝
 
 Interested in writing a new policy?
-  1. Checkout the step-by-step tutorials inside of our [documentation](https://docs.kubewarden.io) 📖
-  1. Add [`kwctl`](https://github.com/kubewarden/kwctl) to your toolbox 🛠️ 🧰
-  1. Pick one of the languages from below
-
-| Language | Project Template | SDK | Validation | Mutation | Maturity |
-|----------|------------------|-----|------------|----------|----------|
-| Rust     | [:octocat:](https://github.com/kubewarden/rust-policy-template) | [:octocat:](https://github.com/kubewarden/policy-sdk-rust) | ✔️ | ✔️ | 🔝 |
-| Go ([TinyGo](https://tinygo.org/)) | [:octocat:](https://github.com/kubewarden/go-policy-template) | [:octocat:](https://github.com/kubewarden/policy-sdk-go) | ✔️ | ✔️ | ↗️ |
-| Swift | [:octocat:](https://github.com/kubewarden/swift-policy-template) | [:octocat:](https://github.com/kubewarden/policy-sdk-swift) | ✔️ | ✔️ | ↗️ |
-| Rego - Open Policy Agent | [:octocat:](https://github.com/kubewarden/opa-policy-template) | [Rego built-ins](https://www.openpolicyagent.org/docs/latest/policy-reference/#built-in-functions) | ✔️ | ❌ | 🔝 |
-| Rego - Gatekeeper | [:octocat:](https://github.com/kubewarden/gatekeeper-policy-template) | [Rego built-ins](https://www.openpolicyagent.org/docs/latest/policy-reference/#built-in-functions) | ✔️ | ❌ | 🔝 |
-| DotNet     |❌  | [:octocat:](https://github.com/kubewarden/policy-sdk-rust) | ✔️ | ✔️ | ↗️  |
 
+Kubewarden allows you to write policies using a variety of programming languages, including Rust, Go, Rego, CEL and others.
 
-Can't find your favorite language? 🔍 Reach out to us and let's have a chat!
+These are some useful resources to get you started:
+ - [Step-by-step tutorials](https://docs.kubewarden.io/tutorials/writing-policies) inside of our documentation 📖
+ - [Policy SDK](https://github.com/kubewarden/community/#policies-sdks): overview of the Policy SDKs available, and their maturity level 📚
+ - [Policy templates](https://github.com/kubewarden/community/#policies-templates): boilerplate code to help you get started 🚀  
```

Comment:
```suggestion

- [Step-by-step tutorials](https://docs.kubewarden.io/tutorials/writing-policies) inside of our documentation 📖
- [Policy SDKs](https://github.com/kubewarden/community/#policies-sdks): overview of the Policy SDKs available, and their maturity level 📚
- [Policy templates](https://github.com/kubewarden/community/#policies-templates): boilerplate code to help you get started 🚀 
```

- adjust spacing to pass `markdownlint` (new line before the list and no space before the hyphen)
- "SDK" -> "SDKs"